### ISSUE TITLE
clarify Django deployment and SCRIPT_NAME

### ISSUE THREAD

**Issue by ola-t** [evansd/whitenoise#140]
Thanks for great solution, althoguth I came across issue with deploying Django app under custom prefix. Following the doc it sounded like cutom prefix will be handled automaticaly, althought it took me a while to debug that statics have to be configured to the following:

```
WHITENOISE_STATIC_PREFIX = '/static/'
FORCE_SCRIPT_NAME = '/myapp'
STATIC_URL = '%s%s' % (FORCE_SCRIPT_NAME, WHITENOISE_STATIC_PREFIX)
```

see also https://stackoverflow.com/questions/45557034/django-static-url-when-script-name-set

**Comment by evansd** [comment]
Yes, you're right that this process wasn't as smooth as it could be. I've made some changes (c72ecd37a0688627936be911268ea9f36569a4a7) so that the  `FORCE_SCRIPT_NAME`  setting is automatically used if it's available, and I've also updated the [docs](http://whitenoise.evans.io/en/latest/django.html#deploying-an-application-which-is-not-at-the-root-of-the-domain).

These changes will be released as part of version 4.0.
